dcc-portal-ui/app/scripts/downloader/js/request.js

```diff
@@ -136,8 +136,10 @@
 
         $scope.params.processing = false;
       },function (error) {
-        $scope.params.processing = false;
-        $scope.params.downloadEnabled = false;
+        if(error.status === 503){
+          $scope.params.processing = false;
+          $scope.params.downloadEnabled = false;
+        }
```

Comment:
I'm not sure if I'm understanding correctly, but I think our app's `setErrorInterceptor` ([here](https://github.com/icgc-dcc/dcc-portal/blob/41fe22825c3c1edef43d6dc53315ac558f606bc1/dcc-portal-ui/app/scripts/app/js/app.js#L602-L614)) would receive the error before this error handler does

I think we may want to change its current check of `if (response.status === 500) {` to `if (_.inRange(response.status, 500, 600)) {`